dts/bindings/sensor/sensirion,sgp40.yaml

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2021, Leonard Pollak
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Sensirion SGP40 Multipixel Gas Sensor
+
+compatible: "sensirion,sgp40"
+
+include: i2c-device.yaml
+
+properties:
+    enable-selftest:
+      type: boolean
+      description: |
+        Enabling this will run a selftest when the driver initializes.
```

Comment:
Is this a hardware or software feature?  If its a pure software feature this should be a Kconfig option and not a devicetree one.

Reply:
the selftest is a specific functionality of this hardware.

Reply:
+1 on DT, may be enabled on certain instances only